apps/aecontract/src/aect_contracts.erl

```diff
@@ -89,7 +77,7 @@
 -define(PUB_SIZE, 32).
 -define(HASH_SIZE, 32).
 -define(CONTRACT_TYPE, contract).
--define(CONTRACT_VSN, 1).
+-define(CONTRACT_VSN, 2).
```

Comment:
I expect you can keep 1 (cc @gorillainduction ) as anyway we [plan to bump consensus](https://www.pivotaltracker.com/story/show/157506836)